_docs/upgrading-portus.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Upgrading Portus
+layout: post
+order: 6
+longtitle: How to upgrade Portus across diferent versions
+---
+
+## Upgrading from 2.0 to 2.1
+
+### Changes on the configuration
+
+First of all, one of the breaking changes from 2.0 to 2.1 is that the machine
+FQDN is no longer considered a secret, but a mere configuration value. In order
+to provide a smoother transition, in Portus 2.0.5 we allow the FQDN to be
+specified in either the `config/secrets.yml` file or as a proper configurable
+value (see [this](https://github.com/SUSE/Portus/commit/f0850459cc43e9b9258e70867d5608f2ef303f3e) commit).
+This configurable option is defined like this:
+
+{% highlight yaml %}
+machine_fqdn:
+  value: "portus.test.lan"
+{% endhighlight %}
+
+Therefore, just provide the same value as you had from the `config/secrets.yml`
+file, into the `config/config-local.yml` file or the `PORTUS_MACHINE_FQDN_VALUE`
+environment variable.
+
+Another breaking change is the `jwt_expiration_time` configuration option. Now,
+instead of being a standalone option, it's been put inside of the `registry`
+configuration. So, now instead of:
+
+{% highlight yaml %}
+jwt_expiration_time
+  value: "5.minutes"
+{% endhighlight %}
+
+The current configuration is like this:
+
+{% highlight yaml %}
+registry:
+  jwt_expiration_time:
+    value: 5
+{% endhighlight %}
+
+Moreover, as you can see, the value of `jwt_expiration_time` has been
+simplified. Now, instead of `"5.minutes"` you have to specify it like this
+`5`. The expiration time is now represented in minutes, so it's no longer needed
+to specify it like that. The `"5.minutes"` format is still supported, but this
+support will be removed in the future.
+
+Besides all that, there have been lots of additions in the configuration that
+you can check out [here](/docs/Configuring-Portus.html).
+
+### Upgrading the database
+
+<div class="alert alert-info">
+  <strong>Important note</strong>: before doing anything at all, make sure to
+  backup the contents of the database.
+</div>
+
+
+With the new code in place, simply run the following (wrap it with `portusctl`
+if you are using the RPM for openSUSE/SLE):
+
+```
+$ rake db:migrate
+$ rake migrate:update_personal_namespaces
+```
+
+After that, if you are using an LDAP server for authentication, run the following:
+
+```
+$ rake migrate:update_ldap_names
+```
+
+The previous task will list you the users that will be updated from it, and then
+it will ask you whether you want to proceed or not. If you don't want this task
+to ask you this question (make it non-interactive), simply set the
+`PORTUS_FORCE_LDAP_NAME_UPDATE` environment variable before running the rake
+task (note that the task will only check whether this environment variable is
```

Comment:
Rephrase: "Note that it will only check the existence of the environment variable, not its value."